gammapy/modeling/models/spatial.py

```diff
@@ -775,6 +775,28 @@ def to_region(self, **kwargs):
             **kwargs,
         )
 
+    @classmethod
+    def from_region(cls, region, **kwargs):
+        """Create a `DiskSpatialModel from a ~regions.EllipseSkyRegion`
+
+        Parameters
+        ----------
+        region : `~regions.EllipseSkyRegion`
+            region to create model from
+
+        Returns
+        -------
+        spatial_model : `~gammapy.modeling.models.DiskSpatialModel`
+        """
+        frame = kwargs.pop("frame", region.center.frame)
+        model = cls.from_position(getattr(region.center, frame))
+        r0 = max(region.width, region.height)
```

Comment:
I think in regions there is no convention on which is the major or minor axis, so this could potential flip the ellipse by 90 deg?

Reply:
Thank you @adonath ! Indeed my bad. fixed it now